### PR TITLE
Clickable build

### PR DESCRIPTION
**Pull request by jezek**
With these changes the messaging app can be build using clickable.

~~_NOTE:_ As for now, building with clickable fails, due to libnotify dependencies missing in clickable/ubuntu-sdk:16.04-armhf docker image.
[Issue is filled](https://gitlab.com/clickable/clickable/issues/100), but in the meantime you can install the ```libnotify-dev:armhf``` by yourself into the image and then use it via the ```--docker-image``` parameter.~~

_EDIT:_ Building app with clickable now works as expected. If the libnotify error occurs, use ```clickable update``` and then try again.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,19 @@ ReadMe - Ubuntu Messaging App
...
+=======================
+Install [clickable](http://clickable.bhdouglass.com/en/latest/), then app then can be build and deployed by simply running:
+
+```
+clickable
+```
+
+For faster build speeds, building app tests is disabled in ```clickable.json``` 
+
+_NOTE:_ As for now, building with clickable fails, due to libnotify dependencies missing in clickable/ubuntu-sdk:16.04-armhf docker image.
```
Comment (UniversalSuperBox) [inline README.md]: If this issue is resolved, please update the README

**Comment by bhdouglass** [comment]
Does this mean we could ship updates to the messaging app via the openstore? What else needs to happen for that?

File: README.md (hunk 1 above)
Comment (jezek) [reply README.md]: Updated in https://github.com/ubports/messaging-app/pull/73/commits/63ff1c433fb016f853a0901db68589da0bfd672e commit

**Comment by jezek** [comment]
> Does this mean we could ship updates to the messaging app via the openstore? What else needs to happen for that?

- Texts in ```messaging-app.desktop``` file are not translated using clickable. (I didn't bother, cause I thought clickable version will be only for testing & developing)
- App version in ```click/manifest.json.in``` is not not consistent with version in snapcraft manifest (I could not find any other version info). 
- Bindings from other apps (contacts, notifications,...) have to point to right messaging-app
- Apparmor is configured for full access (Maybe this isn't a drawback)

Edit: added items to list

**Comment by UniversalSuperBox** [comment]
It doesn't seem like this can be shipped from the store yet. However the push notifications are handled for text/mms is not functioning. Messages are only notified upon opening the app.

**Comment by UniversalSuperBox** [comment]
If you're okay with everything here (including the versioning format), merge when ready @bhdouglass

**Comment by bhdouglass** [comment]
Seems like this is a good first step towards shipping a click version of the messaging app :+1:

**Pull request merged**